test/action.yaml

```diff
@@ -15,6 +15,10 @@ inputs:
     required: false
     description: "Relative or absolute path to the terraform cli config file"
     default: ./.terraformrc
+  use-terraform-wrapper:
+    required: false
+    description: "Whether or not to use terraform wrapper when setting up terraform"
+    default: false
```

Comment:
I'm guessing this is giving you a warning in your editor?

Reply:
Good catch. IDEA is showing a hollow checkmark rather than a solid green check mark. Didn't notice the difference. I did just add quotes around the default boolean values.